values.yaml

```diff
@@ -120,4 +106,29 @@ relay:
 
   ## Secrets don't like '-'
   secrets:
-    relayCogToken: 
+    relayCogToken:
+
+nginx:
+  name: nginx
+  replicaCount: 1
+  image:
+    repository: nginx
+    tag: 1.11
+  resources:
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    requests:
+      cpu: 100m
+      memory: 128Mi
+  serviceType: LoadBalancer
+  ingress:
```

Comment:
nit: Does this belong under `nginx`?

Saying "I'd like to turn on an Ingress for the nginx service" vs saying "I'd like to turn on an Ingress for my Cog (instance|chart)"